src/ontology/uberon-edit.obo

```diff
@@ -222238,6 +222238,17 @@ synonym: "fifth molar tooth" EXACT []
 is_a: UBERON:0003655 ! molar tooth
 relationship: dc-contributor https://orcid.org/0000-0003-2699-3066
 
+[Term]
+id: UBERON:7500127
+name: otion
+def: "The most lateral anatomical point on the mastoid process of temporal bone." [http://desinfos.eu/archeozoologie.desinfos.eu/PERSO/Aides%20determination/Osteometrie/Von%20Den%20Driesch\,%201976\,%20A%20guide%20to%20the%20measurement%20of%20animal%20bones%20from%20archaeological%20sites.pdf, ISBN:9780873659505]
+is_a: UBERON:0006983 ! anatomical point
+relationship: dc-contributor https://orcid.org/0000-0001-8415-4848
+relationship: dc-contributor https://orcid.org/0000-0003-2699-3066
+relationship: part_of UBERON:0011220 ! mastoid process of temporal bone
+property_value: depicted_by "ISSN 0372-5480, Fig. 1, \"Ot\" (https://intranet.vef.hr/vetarhiv/papers/2014-84-4-6.pdf)" xsd:string
```

Comment:
While "Ot" likely represents otion based on the other references, this document itself does not appear to have a legend defining "Ot" as otion and does not make mention of it in the text.